lib/screens/department_screen/department_constants.dart

```diff
@@ -0,0 +1,6 @@
+const String ABOUT_SECTION = "About";
+const String FACULTY_DETAILS_SECTION = "Faculty Details";
+const String PLACEMENT_DETAILS_SECTION = "Placement Details";
+const String RESULT_ANALYSIS_SECTION = "Result Analysis";
+const String CURRICULUM_SECTION = "Curriculum";
+const String MORE_SECTION = "More";
```

Comment:
You don't need this? It is not required

Reply:
Requirement for this will go away if we use enums.

New see the new update in dart 2.15 it has better enum support